squad/frontend/views.py

```diff
@@ -446,11 +446,10 @@ def test_run_suite_tests(request, group_slug, project_slug, build_version, testr
     )
 
     all_tests = context['status'].tests.prefetch_related(
-        'suite',
         'metadata',
         'known_issues',
         'suite__metadata'
-    ).order_by(Case(When(result=False, then=0), When(result=True, then=2), default=1), 'name')
+    ).order_by(Case(When(result=False, then=0), When(result=True, then=2), default=1), 'metadata__name')
```

Comment:
here you are sorting on `metadata__name`! :)

Reply:
I guess this one was the correct one!